docker-install.sh

```diff
@@ -64,4 +64,4 @@ run_cmd "make install" make -j install
 
 run_cmd "remove temp packages" apt-get remove -y $TEMP_PACKAGES
 run_cmd "remove unused packages" apt-get autoremove -y
-run_cmd "remove tmp files" rm -rf /tmp/*
+run_cmd "remove tmp files" rm -rf /tmp/* ${ser2net_archive}
```

Comment:
The archive should not be deleted since the path they are stored in is a shared docker volume, only available during builds to ensure building releases don't download files over and over

Reply:
ah yes, you're right, will revert that one!